python/tests/unit/connectors/open_ai/contents/conftest.py

```diff
@@ -0,0 +1,14 @@
+from pytest import fixture
+
+from semantic_kernel.connectors.ai.open_ai.contents.function_call import FunctionCall
+from semantic_kernel.connectors.ai.open_ai.contents.tool_calls import ToolCall
+
+
+@fixture(scope="module")
+def fc():
```

Comment:
Could we have a better name? Same for `tc`?

Reply:
good point, thanks